test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -210,7 +210,7 @@ export default class BaseFormObject {
 
   getErrorBoxText() {
     if (userVariables.v3) {
-      return within(this.el).getByRole('alert').innerText;
+      return within(this.el).getAllByRole('alert').nth(0).innerText;
```

Comment:
Is this the change that allows you to get the correct error box? Seems like `getByRole` would default to the first match